playbooks/roles/common-server-init/templates/netplan/floating-ip.yaml

```diff
@@ -0,0 +1,12 @@
+network:
+  version: 2
+  renderer: networkd
+  ethernets:
+    {{ COMMON_FLOATING_INTERFACE_NAME }}:
+      dhcp4: yes
+      dhcp6: yes
```

Comment:
@giovannicimolin, with DHCP enabled here, and a static IP set in the `addresses` list below, does netplan add the IP address received from DHCP (which is also a static IP) to the same interface without overwriting or affecting the floating IP address?

Reply:
Yes. Netplan accepts this kind of setup even without using the `label` attributes that I've put.